.github/CONTRIBUTING.md

```diff
@@ -8,20 +8,55 @@ Afterwards, you can either contribute icons or code.
 ## Contributing icons
 To contribute icons, you only need an icon editor, a file explorer, a text editor, and a terminal window.
 
-### Icon guidelines
-See the below image for a summary of the icon guidelines. If you don't follow them, a team member will likely request changing the icons.
+### Creating Icons 1/5 — Artboard & Grid
 
-![](images/contributing-image-1.png)
+![image](https://github.com/LawnchairLauncher/lawnicons/assets/49114212/4b037c67-7349-4e21-8100-a46a91045123)
```

Comment:
Please upload *all* images to the repository. Using external image sources isn't recommended (even in the same repo), because it isn't stable (the image can be deleted later or something, not available offline, etc).

You can use custom names or follow the existing image format (`contributing-image-#`).